lib/test_mock.py

```diff
@@ -1,17 +1,17 @@
 #!/bin/python
 import sys
 from hardware_services.hardware_service import get_hardware_service, set_hardware_service
-from hardware_services.hardware_drivers.mock_driver import MockDriver
+from hardware_services.hardware_drivers.hil_driver import HilDriver
```

Comment:
Why are we doing this in the test environment?  Wouldn't we always want to use Mock here?